### PR TITLE
Add the first M3 pages

### PR DESCRIPTION
**Pull request by lornajane**
M3 will be one of the products that is on the devportal platform at launch. This PR is the work-in-progress for adding its content.

[x] Welcome page
[x] Getting started (including hooking up your existing service)
[x] Guide to M3DB components (with diagram)
[x] Guide to aggregations and scaling
[x] Tutorial on viewing M3DB data with Grafana
[x] Tutorial on writing to M3DB with Telegraf
[x] Code samples for Python (exists in aiven-examples)
[x] Code samples for PHP
[x] Code samples for Go

### CODE REVIEW

File: docs/products/m3db/concepts/scaling-m3.rst [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+* number of time series
+
+* number of points within single block
+
+* number of configured namespaces
+
+For efficient on-disk storage, M3's default recommendation is to have about 720 points per block. So for 15 seconds per point case, that would indicate 180 minutes, or 3 hours per block.  Following the recommendation may be worth it for high resolution namespaces, but for longer lived ones it may lead to too much data being kept in memory. For example, if there is a point per day, closing block every two years would not work. In that sort of case, e.g. day(s) or at most week's block size duration is recommended.
+
+The data within current block (and for a bit longer depending on buffer past duration) is stored in RAM until they are flushed to disk, and this is multiplied by number of time series AND it is also multiplied by number of namespaces that contain those particular points.
```
Comment (ftisiot) [inline docs/products/m3db/concepts/scaling-m3.rst:121]: What is buffer past duration? Is it necessary to specify it now?

Comment (ftisiot) [reply docs/products/m3db/concepts/scaling-m3.rst:121]: "this is multiplied by number of time series AND it is also multiplied by number of namespaces that contain those particular points"
->
"this is multiplied by number of time series AND by the number of namespaces containing those particular points"

File: docs/products/m3db/concepts/scaling-m3.rst [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+Given a consistent amount of points per second coming in, the memory usage can be reduced by:
+
+* having smaller block size duration (disk usage will grow faster)
+
+* having fewer namespaces
+
+* filtering data that gets to aggregated namespaces 
+
+* configuring shorter buffer past duration as data for current block is retained in memory that much longer after the block is already past; Aiven for M3 defaults to maximum value, which is block size - 1 second); note that if changing buffer past duration to e.g. 1 minute, data older than 1 minute will be simply discarded
```
Comment (ftisiot) [inline docs/products/m3db/concepts/scaling-m3.rst:131]: Adding a note on what buffer past duration does is necessary

File: docs/products/m3db/concepts/scaling-m3.rst [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+For M3DB memory usage cost, using the disk example, what is in RAM per series is (at least)::
+
+    (U) 4 * 60 * 2 = 480 points +
```
Comment (ftisiot) [inline docs/products/m3db/concepts/scaling-m3.rst:135]: why the 4, 60, 2? 2 Is the hour, 60 is the minutes, 4?

Comment (ftisiot) [reply docs/products/m3db/concepts/scaling-m3.rst:135]: We need to give more info on how the math is performed

File: docs/products/m3db/concepts/scaling-m3.rst [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+    (A) 4 * 60 * 4 = 960 points +
+
+    (B) 60 * 4 = 240 points +
+
+    (C) 12 = 12 points
+
+    =~ 1700 points + whatever the not-quite-insignificant per-time-series overhead is (* replication factor)
```
Comment (ftisiot) [inline docs/products/m3db/concepts/scaling-m3.rst:143]: "whatever the not-quite-insignificant per-time-series overhead is (* replication factor)" -> time-series point overhead * replication_factor

File: docs/products/m3db/concepts/scaling-m3.rst [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+M3Aggregator will also hold subset of this - unaggregated namespace (U) will not be included.  And this is for single time series; typically you may have more than that (we do have tens of millions at the time of this article's writing).
```
Comment (ftisiot) [inline docs/products/m3db/concepts/scaling-m3.rst:145]: M3Aggregator also stores a subset of the points - excluding unaggregated namespace (U) - for its aggregation purposes.

Comment (ftisiot) [reply docs/products/m3db/concepts/scaling-m3.rst:145]: "And this is for single time series; typically you may have more than that (we do have tens of millions at the time of this article's writing)."
->
And this memory figures are for single time series; Typically you may have more than that (we do have tens of millions at the time of this article's writing).

File: docs/products/m3db/concepts/scaling-m3.rst [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+
+
+Scaling Recommendations
```
Comment (ftisiot) [inline docs/products/m3db/concepts/scaling-m3.rst:148]: "Scaling general Recommendations" as title?

File: docs/products/m3db/concepts/scaling-m3.rst [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+Real-world example: Aiven production configuration
+--------------------------------------------------
+
+At the moment we use with 30 second typical scrape interval with following namespace configuration:
+
+* 2 day unaggregated namespace, and 
+
+* 1 month aggregated namespace with 10min resolution
+
+and even that sort of aggregation configuration causes CPU and memory usage to be significantly more in M3DB than if we just had (longer) unaggregated namespace and no aggregation in use. However, our disk usage is only less than that of 4 days' worth of unaggregated data and the aggregated data performs much better for historic queries (e.g. looking at graph for one week).
```
Comment (ftisiot) [inline docs/products/m3db/concepts/scaling-m3.rst:162]: We might need to rephrase this

File: docs/products/m3db/getting-started.rst [hunk 8]
```diff
@@ -0,0 +1,28 @@
+Getting Started
+===============
+
+There are two M3-related services available in the `Aiven console <https://console.aiven.io>`_. Get started with an **M3DB** service; this is the part that stores data.
+
+.. note::
+   The M3 Aggregator service provides extra query functionality when performing complex queries against the distributed data set
+
+Choose your cloud provider and location to deploy to, then choose which plan to use.
```
Comment (ftisiot) [inline docs/products/m3db/getting-started.rst:9]: location -> region?

File: docs/products/m3db/getting-started.rst [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+If you're just trying out M3DB, a single-node setup is available with our startup plans. This isn't recommended for production use however; for that you should try the 3-node clusters on the business plans, or a premium plan to meet your needs.
```
Comment (ftisiot) [inline docs/products/m3db/getting-started.rst:11]: Should we mention that you can upscale downscale when needed? Is this happening while the service is online for M3?

File: docs/products/m3db/howto/grafana.rst [hunk 10]
```diff
@@ -0,0 +1,37 @@
+Visualise M3DB Data with Grafana
+================================
+
+Since M3DB is best for time series data, many individual metrics, then it's nicer to visualise the data than try to view it in a table or log. Luckily, Aiven can set up the Grafana and the integration between the two services for you.
```
Comment (ftisiot) [inline docs/products/m3db/howto/grafana.rst:4]: "many individual metrics" -> "composed by many individual metrics"

File: docs/products/m3db/howto/grafana.rst [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+Integrate M3DB and Grafana
+--------------------------
+
+1. On the service overview page for your M3DB service, go to "Manage Integrations" and choose the "Dashboard" option.
+
+2. Choose either a new or existing service.
+
+   - A new service will ask you to confirm the cloud, region and plan to use. You should also give your service a name. The service overview page shows the nodes rebuilding, and then indicates when they are ready.
```
Comment (ftisiot) [inline docs/products/m3db/howto/grafana.rst:13]: "select" instead of "confirm"

Comment (ftisiot) [reply docs/products/m3db/howto/grafana.rst:13]: Should we state something about selecting the same cloud provider and region for minimal latency?

File: docs/products/m3db/howto/telegraf.rst [hunk 12]
```diff
@@ -0,0 +1,51 @@
+How to Write to M3 from Telegraf
+================================
+
+M3 supports multiple methods of writing metrics to the M3 database. Two common options are the InfluxDB wire-protocol, and the Prometheus remote write protocol. It is possible to write metrics from an existing Telegraf setup to M3 using either of these protocols, so we've covered the configuration for both options here.
+
+Configuring Telegraf InfluxDB output plugin for M3
+--------------------------------------------------
+
+This is how you configure Telegraf to send metrics to M3 using the InfluxDB line-protocol. This configuration needs to be part of the output plugins configuration in your Telegraf configuration file::
```
Comment (ftisiot) [inline docs/products/m3db/howto/telegraf.rst:9]: "This **configuration** needs to be part of the output plugins **configuration** in your Telegraf **configuration** file::" 
->
"The following lines need to be part of the output plugins section in your Telegraf configuration file::"

File: docs/products/m3db/howto/telegraf.rst [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+
+      ## HTTP Content-Encoding for write request body, can be set to
+      ## "gzip" to compress body or "identity" to apply no encoding.
+
+      content_encoding = "gzip"                                
+
+Configuring Telegraf Prometheus remote write for M3
+---------------------------------------------------
+
+This is how you configure Telegraf send metrics to M3 using the Prometheus remote write protocol. This configuration needs to be part of the output plugins configuration in the Telegraf configuration file::
```
Comment (ftisiot) [inline docs/products/m3db/howto/telegraf.rst:32]: Same as above "This configuration needs to be part of the output plugins configuration in the Telegraf configuration file" 
->
"The following lines need to be part of the output plugins section in your Telegraf configuration file"

File: docs/products/m3db/howto/write-go.rst [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+--------------------------
+
+This example writes some data to an M3DB service from Go, making use of the Prometheus write features.
+
+Variables
+'''''''''
+
+These are the placeholders you will need to replace in the code sample:
+
+==================      ==========================================================
```
Comment (ftisiot) [inline docs/products/m3db/howto/write-go.rst:11]: Should we always use the `list-table` structure?

File: docs/products/m3db/howto/write-go.rst [hunk 15]
```diff
@@ -0,0 +1,42 @@
...
+1. The Prometheus client for Go::
+
+    go get -u github.com/m3db/prometheus_remote_client_golang/promremote
+
+ 
+
+Code
+''''
+
+Add the following to ``main.go`` and replace the placeholder with the Prometheus(write) URL:
```
Comment (ftisiot) [inline docs/products/m3db/howto/write-go.rst:31]: "replace the placeholder with the Prometheus(write) URL" -> "replace the placeholder with the Prometheus(write) URL you can find in the M3DB service overview"

File: docs/products/m3db/howto/write-php.rst [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+---------------------------
+
+This example writes some data to an M3DB service from PHP, making use of the InfluxDB library.
+
+Variables
+'''''''''
+
+These are the placeholders you will need to replace in the code sample:
+
+=================       =======================================================
```
Comment (ftisiot) [inline docs/products/m3db/howto/write-php.rst:11]: Should we always use the `list-table` structure?

File: docs/products/m3db/howto/write-php.rst [hunk 17]
```diff
@@ -0,0 +1,47 @@
...
+
+
+.. note::
+
+    M3DB supports InfluxDB (v1) protocol so we can use the existing library for this database too.
+
+Code
+''''
+
+Add the following to ``index.php`` and replace the placeholders with values for your project:
```
Comment (ftisiot) [inline docs/products/m3db/howto/write-php.rst:36]: which are the placeholders? Maybe we should specify them (SERVICE_HOST, SERVICE_PORT, AVNADMIN_PASS)

File: docs/products/m3db/howto/write-python.rst [hunk 18]
```diff
@@ -0,0 +1,48 @@
...
+------------------------------
+
+This example writes some data to an M3DB service from Python, making use of the InfluxDB library.
+
+Variables
+'''''''''
+
+These are the placeholders you will need to replace in the code sample:
+
+=================       =======================================================
```
Comment (ftisiot) [inline docs/products/m3db/howto/write-python.rst:11]: Should we always use the `list-table` structure?

File: docs/products/m3db/howto/write-python.rst [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+    pip install influxdb
+
+.. note::
+
+    M3DB supports InfluxDB (v1) protocol so we can use the existing library for this database too.
+
+Code
+''''
+
+Add the following to ``main.py`` and replace the placeholders with values for your project:
```
Comment (ftisiot) [inline docs/products/m3db/howto/write-python.rst:37]: which are the placeholders? Maybe we should specify them (SERVICE_HOST, SERVICE_PORT, AVNADMIN_PASS)

File: docs/products/m3db/reference/terminology.rst [hunk 20]
```diff
@@ -0,0 +1,19 @@
+Terminology for M3
+==================
+
+Time series
+    unique series of data identified by a set of labels, containing value(s) for specific points in time
+
+Point
+    (time, value) entry in single time series
```
Comment (ftisiot) [inline docs/products/m3db/reference/terminology.rst:8]: Should we also add that it contain labels?

File: docs/products/m3db/reference/terminology.rst [hunk 21]
```diff
@@ -0,0 +1,19 @@
...
+==================
+
+Time series
+    unique series of data identified by a set of labels, containing value(s) for specific points in time
+
+Point
+    (time, value) entry in single time series 
+
+Namespace
+    retention-period limited view of the database, with (most of the time) all time series included; either unaggregated (all points), or aggregated at some resolution (at most only one point per resolution duration)
```
Comment (ftisiot) [inline docs/products/m3db/reference/terminology.rst:11]: I would rephrase this:

"analogous to tables in other databases, contain a view of time series dataset limited to the retention-period". It can be **unaggregated** (all points), or **aggregated** at some resolution (at most only one point per resolution duration)

File: docs/products/m3db/reference/terminology.rst [hunk 22]
```diff
@@ -0,0 +1,19 @@
...
+
+Replication factor
+    M3 stores data redundantly (if replication factor > 1), and uses by default quorum writes and best-effort quorum reads. In Aiven for M3, replication factor is hardcoded at 3, which means that single availability zone's worth of nodes can be lost and database still stays functional.
+
+Shard
+    M3 databases have defined number of shards, and all namespaces within M3 are split also to the shards and replicated based on replication factor. In Aiven for M3, shard count is hardcoded at 60 but can be changed by request for particular service (however, all data is lost if the count is changed).
```
Comment (ftisiot) [inline docs/products/m3db/reference/terminology.rst:17]: This describes that M3 is using sharding but doesn't say what sharding is. We could make the parallel with Kafka partitions?

File: docs/products/m3db/concepts/scaling-m3.rst (hunk 3 above)
Comment (ftisiot) [reply docs/products/m3db/concepts/scaling-m3.rst:135]: 60 looks like is the default for shards (according to our documentsion)

**Review by ftisiot — CHANGES_REQUESTED** [review]
Great Content! I love how everything is linked as well!

Few notes:
* Most of the articles are in a great shape with minor comments
* The article with most comments is the one about sizing (i believe it was an extract from Markus internal doc), needs some parsing because it's information heavy and has few external references which are missing

File: docs/products/m3db/concepts/m3-components.rst [hunk 23]
```diff
@@ -0,0 +1,22 @@
+Aiven for M3 Components
+=======================
+
+M3 consists of three components, which at Aiven are provided as two separate services:
```
[the remark this answers is not in the document]
Comment (lornajane) [reply docs/products/m3db/concepts/m3-components.rst:4]: It does say "two separate services" though ... I think this is OK.

File: docs/products/m3db/concepts/namespaces-aggregation.rst [hunk 24]
```diff
@@ -0,0 +1,36 @@
...
+
+Unaggregated Namespace
+----------------------
+
+Incoming data points are written to the unaggregated namespace. It's the default table, if you like.
+
+Aggregated Namespace
+--------------------
+
+An aggregated namespace has some data, usually downsampled to a lower resolution, and retained for a different period of time.
```
[the remark this answers is not in the document]
Comment (lornajane) [reply docs/products/m3db/concepts/namespaces-aggregation.rst:33]: I am also unsure about the aggregate function selection ...

File: docs/products/m3db/getting-started.rst (hunk 8 above)
Comment (lornajane) [reply docs/products/m3db/getting-started.rst:9]: Yes, thanks!

File: docs/products/m3db/getting-started.rst (hunk 9 above)
Comment (lornajane) [reply docs/products/m3db/getting-started.rst:11]: I'm not sure about this so I didn't include it yet.

Comment (lornajane) [reply docs/products/m3db/getting-started.rst:11]: Oh, you should be able to upscale, I should at least say that!

File: docs/products/m3db/howto/grafana.rst (hunk 10 above)
Comment (lornajane) [reply docs/products/m3db/howto/grafana.rst:4]: I went with "consisting of" but yes, good catch

File: docs/products/m3db/howto/grafana.rst (hunk 11 above)
Comment (lornajane) [reply docs/products/m3db/howto/grafana.rst:13]: In general, the howto content types should have as little extra explanation as possible. Maybe we should have an explainer for cloud regions or something though if you think this is important and not already well understood.

File: docs/products/m3db/howto/write-go.rst (hunk 14 above)
Comment (lornajane) [reply docs/products/m3db/howto/write-go.rst:11]: I'm not sure it matters (this might bite us later though, you're quite right!)

File: docs/products/m3db/howto/write-go.rst (hunk 15 above)
Comment (lornajane) [reply docs/products/m3db/howto/write-go.rst:31]: Hmm, it says that in the previous section. Maybe I just should say "placeholders"

File: docs/products/m3db/howto/write-php.rst (hunk 17 above)
Comment (lornajane) [reply docs/products/m3db/howto/write-php.rst:36]: They are described on lines 11-17 ... does this answer your question?

File: docs/products/m3db/howto/write-python.rst (hunk 19 above)
Comment (lornajane) [reply docs/products/m3db/howto/write-python.rst:37]: They are detailed above, with description information for each.

File: docs/products/m3db/reference/terminology.rst (hunk 20 above)
Comment (lornajane) [reply docs/products/m3db/reference/terminology.rst:8]: Good idea

File: docs/products/m3db/reference/terminology.rst (hunk 21 above)
Comment (lornajane) [reply docs/products/m3db/reference/terminology.rst:11]: adapted slightly but thanks!

File: docs/products/m3db/concepts/scaling-m3.rst [hunk 25]
```diff
@@ -0,0 +1,162 @@
...
+      - 4 weeks
+      - 1 min
+      - 4 hours
+    * - Aggregated C (C)
+      - 8 weeks
+      - 1 hour
+      - 12 hours
+
+
+If we assume that there are 15 seconds per point in each time series, then we can look at the storage by week since the reading was taken:
```
[the remark this answers is not in the document]
Comment (lornajane) [reply docs/products/m3db/concepts/scaling-m3.rst:51]: This is a great question and the calculations for week 1 below look like it expects the same number of points in both the unaggregated namespace and namespace A

File: docs/products/m3db/concepts/namespaces-aggregation.rst [hunk 26]
```diff
@@ -0,0 +1,36 @@
+About M3DB Namespaces and Aggregation
+=====================================
+
+M3DB has a concept of namespaces.
```
[the remark this answers is not in the document]
Comment (lornajane) [reply docs/products/m3db/concepts/namespaces-aggregation.rst]: That's a nice analogy, added

File: docs/products/m3db/concepts/m3-components.rst [hunk 27]
```diff
@@ -0,0 +1,22 @@
...
+
+* **M3DB** consists of ``X`` number of M3DB + M3Coordinator pairs. Where ``X`` is the number of nodes in the plan you choose.
+* **M3 Aggregator** is an optional additional component that you need if you use aggregated namespaces, the aggregator does the downsampling. Again, the plan you choose dictates how many of these you have.
```
[the remark this answers is not in the document]
Comment (lornajane) [reply docs/products/m3db/concepts/m3-components.rst:7]: Plan - yes, could be clearer. I'm not sure about overexplaining things that are common to all services.

File: docs/products/m3db/concepts/scaling-m3.rst (hunk 25 above)
[the remark this answers is not in the document]
Comment (lornajane) [reply docs/products/m3db/concepts/scaling-m3.rst]: I've removed this file completely from the repo, it's just not ready to merge

File: docs/products/m3db/howto/telegraf.rst (hunk 12 above)
Comment (lornajane) [reply docs/products/m3db/howto/telegraf.rst]: Ha, that is a bit difficult to digest! Updated.

**Comment by lornajane** [comment]
I've queried the aggregation function question and removed the scaling doc completely. It's in a branch and can return as a pull request at a later date, as a separate task.

File: docs/products/m3db/concepts/m3-components.rst [hunk 28]
```diff
@@ -0,0 +1,22 @@
...
+
+.. mermaid::
+
+    graph LR
+        App(Your application) ---> M3[(M3DB + M3 Coordinator)]
+        M3 ----> Gr((Grafana))
+        M3 <-.-> Agg(M3 Aggregator)
+
+
+Another common setup is to use Prometheus with M3 as the storage element.
```
Comment (ftisiot) [inline docs/products/m3db/concepts/m3-components.rst:22]: should we add a link to Prometheus?
-> https://prometheus.io/
or
-> https://m3db.io/docs/integrations/prometheus/

File: docs/products/m3db/howto/monitoring.rst [hunk 29]
```diff
@@ -0,0 +1,14 @@
+Monitor Aiven Services with M3DB
+================================
+
+M3DB is a perfect fit for a monitoring platform. It is designed to handle large volumes of metrics, and it's very good at that. Whether you are already working at scale or just curious about M3DB and monitoring, setting up an M3DB to monitor your existing Aiven services (or new ones if you like) is a really nice way to get started with this platform.
+
+Start from the service overview page of the service you would like to monitor with M3. So if you're going to track the metrics of your PostgreSQL service on Aiven, then start on the PostgreSQL service overview page.
```
Comment (ftisiot) [inline docs/products/m3db/howto/monitoring.rst:6]: Should we put in bold **Service Overview** to make it more visible to readers?

File: docs/products/m3db/howto/monitoring.rst [hunk 30]
```diff
@@ -0,0 +1,14 @@
...
+
+1. Under "Manage Integrations", look for "Metrics" (not available on all services yet).
```
Comment (ftisiot) [inline docs/products/m3db/howto/monitoring.rst:8]: Adding the image of the Manage Integration screen will help readers understand if they are in the good place and what to look for?

File: docs/products/m3db/howto/telegraf.rst [hunk 31]
```diff
@@ -0,0 +1,51 @@
+How to Write to M3 from Telegraf
+================================
+
+M3 supports multiple methods of writing metrics to the M3 database. Two common options are the InfluxDB wire-protocol, and the Prometheus remote write protocol. It is possible to write metrics from an existing Telegraf setup to M3 using either of these protocols, so we've covered the configuration for both options here.
+
```
Comment (ftisiot) [inline docs/products/m3db/howto/telegraf.rst:5]: Should we also add here the **Variables** section we have for the other HowTos and reference the variables in the config files?

**Review by ftisiot — COMMENTED** [review]
Few comments, that are more suggestions to keep the content in line with other pages and give proper references to users.

File: docs/products/m3db/concepts/namespaces-aggregation.rst (hunk 24 above)
[the remark this answers is not in the document]
Comment (lornajane) [reply docs/products/m3db/concepts/namespaces-aggregation.rst]: I've removed this page for now

File: docs/products/m3db/concepts/m3-components.rst (hunk 28 above)
Comment (lornajane) [reply docs/products/m3db/concepts/m3-components.rst:22]: Added

File: docs/products/m3db/howto/monitoring.rst (hunk 30 above)
Comment (lornajane) [reply docs/products/m3db/howto/monitoring.rst:8]: I dislike screenshots in general because they are hard to maintain - and the integrations screen definitely should change from time to time so I'm going to leave it out.

File: docs/products/m3db/howto/telegraf.rst (hunk 31 above)
Comment (lornajane) [reply docs/products/m3db/howto/telegraf.rst:5]: Oh we can! It's not a code sample, but why not?

File: docs/products/m3db/howto/monitoring.rst (hunk 29 above)
Comment (lornajane) [reply docs/products/m3db/howto/monitoring.rst:6]: can do

File: docs/products/m3db/howto/telegraf.rst [hunk 32]
```diff
@@ -0,0 +1,63 @@
...
+Variable                Description
+==================      ==========================================================
+``M3_HOST``             The hostname of your M3DB
+``M3_PORT``             Port where M3DB is running
+==================      ==========================================================
+
+Configuring Telegraf InfluxDB output plugin for M3
+--------------------------------------------------
+
+Below is an example of how to configure Telegraf to send metrics to M3 using the InfluxDB line-protocol. These lines belong in the output plugins section of your Telegraf configuration file::
```
Comment (ftisiot) [inline docs/products/m3db/howto/telegraf.rst:21]: Should **output plugins** be in bold?

File: docs/products/m3db/howto/telegraf.rst [hunk 33]
```diff
@@ -0,0 +1,63 @@
...
+      ## The full HTTP URL for your M3 instance.
+      urls = ["https://M3_HOST:M3_PORT/api/v1/influxdb"]
+
+      ## If true, no CREATE DATABASE queries will be sent.
+      ## Even if sent, does not affect M3 in any way.
+      skip_database_creation = true
+      
+      ## HTTP Basic Auth
+      username = "avnadmin"
+      password = "secretpassword"
```
Comment (ftisiot) [inline docs/products/m3db/howto/telegraf.rst:34]: There is a password here, but is not mentioned in the Variables section

File: docs/products/m3db/howto/write-python.rst [hunk 34]
```diff
@@ -0,0 +1,48 @@
...
+'''''''''
+
+These are the placeholders you will need to replace in the code sample:
+
+=================       =======================================================
+Variable                Description
+=================       =======================================================
+``SERVICE_HOST``        Service hostname, found on the service overview page
+``SERVICE_PORT``        Service port number, found on the service overview page
+``AVNADMIN_PASS``       Password for the ``avnadmin`` user
```
Comment (ftisiot) [inline docs/products/m3db/howto/write-python.rst:16]: Should we say that is the default user? "Password for the **default** ``avnadmin`` user"

**Review by ftisiot — APPROVED** [review]
One minor thing to address in the Variables section, and a couple of suggestions. Then is good to go!

File: docs/products/m3db/howto/telegraf.rst (hunk 32 above)
Comment (ftisiot) [reply docs/products/m3db/howto/telegraf.rst:21]: Fixed

File: docs/products/m3db/howto/telegraf.rst (hunk 33 above)
Comment (ftisiot) [reply docs/products/m3db/howto/telegraf.rst:34]: Fixed

File: docs/products/m3db/howto/write-python.rst (hunk 34 above)
Comment (ftisiot) [reply docs/products/m3db/howto/write-python.rst:16]: Fixed
